### PR TITLE
Cypress test automation

### PR DESCRIPTION
**Pull request by nehemiah-abuga**
## PR Summary

In this PR cypress automated tests have been added with the capability to run on CI/CD
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.

## Steps to Validate
- [x] Checkout the branch on your local
-  [x] run ``` npm install ``` to add the dependencies
- [x] run ``` npm run cypress:run_chrome ``` and validate 5 headers tests pass
- [x] run ```npm run cypress:open ``` and select E2E testing and browser of choice. Run the header tests
- [x] In github, navigate to https://github.com/GSA/usagov-benefits-eligibility/actions/workflows/cypress.yml
- [x] Run you tests again the cypress-test-automation branch
- [x] Verify all the 5 headers tests pass

I have verified a success run 
https://github.com/GSA/usagov-benefits-eligibility/actions/runs/5006183114/jobs/8971111047

### CODE REVIEW

**Review by FatmaBakir — APPROVED** [review]
Approved!

**Comment by nehemiah-abuga** [comment]
@scottqueen-bixal  - please review the changes

**Comment by scottqueen-bixal** [comment]
There still seems to be an additional workflow, let's remove the one we don't use
Screenshot 2023-05-24 at 9 48 52 AM

**Review by scottqueen-bixal — CHANGES_REQUESTED** [review]

**Comment by nehemiah-abuga** [comment]
> 



> There still seems to be an additional workflow, let's remove the one we don't use

> This will be removed once we merge to main

File: package.json [hunk 1]
```diff
@@ -66,6 +69,7 @@
     "jest": "^28.1.0",
     "jest-transform-stub": "^2.0.0",
     "lodash": "^4.17.21",
+    "nuxt-i18n": "^6.28.1",
```
Comment (scottqueen-bixal) [inline package.json:72]: this shouldn't be in here

please remove, and rebuild package.json

**Review by scottqueen-bixal — CHANGES_REQUESTED** [review]

**Review by scottqueen-bixal — APPROVED** [review]

**Pull request merged**